benchkit-backend/workload.go

```diff
@@ -20,9 +20,10 @@ package main
 import (
 	"errors"
 	"fmt"
-	"github.com/neo4j/neo4j-go-driver/v5/neo4j"
 	"strconv"
 	"sync"
+
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j"
```

Comment:
```suggestion
	"github.com/neo4j/neo4j-go-driver/v6/neo4j"
```

shouldn't it be v6? I guess `go.mod` needs an update

Reply:
Moving this to another PR 👍